faunadb/query.py

```diff
@@ -199,3 +199,12 @@ def _params(main_params, optional_params):
     if val is not None:
       main_params[key] = val
   return main_params
+
+def _call(name, values):
```

Comment:
I don't really like the name `_call`. It's not apparent just looking at it that it's simply generating a dict for query use. The doc does not clear up this confusion either.